addons/gopaddle-lite/enable

```diff
@@ -0,0 +1,190 @@
+eval set -- "$ARGS"
+
+while true; do
+  case "$1" in
+    -i|--ip)
+      STATICIP="$2"
+      shift 2;;
+    -v|--version)
+      HELM_VERSION="$2"
+      shift 2;;
+    # -s|--storageclass)
+    #   STORAGECLASS="$2"
+    #   shift 2;;
+    --)
+      break;;
+  esac
+done
+
+PUBLICIP=true
+if [ $STATICIP ]
+then
+   echo "static IP of the microk8s cluster: $STATICIP"
+   echo ""
+   "$SNAP/microk8s-enable.wrapper" host-access:ip=$STATICIP
+elif $KUBECTL get node -o yaml | grep ExternalIP >/dev/null 2>&1
+then 
+     echo "Static IP input is not provided. Assuming External IP of the microk8s node for the gopaddle access endpoint."
+     echo ""
+else
+     echo "Static IP input is not provided. External IP is not set for the microk8s node. Assuming Internal IP of the microk8s node for the gopaddle access endpoint."
+     PUBLICIP=false
+     echo ""
+fi
+
+if [ -z "$STORAGECLASS" ]
+then
+  run_with_sudo $KUBECTL apply -f $CURRENT_DIR/storageclass-retain.yaml
+  STORAGECLASS="microk8s-hostpath-gp-retain"
+fi
+
+
+do_prerequisites() {
+  refresh_opt_in_config "authentication-token-webhook" "true" kubelet
+  restart_service kubelet
+  # enable helm3 
+  "$SNAP/microk8s-enable.wrapper" helm3
+
+  # enable storage
+  "$SNAP/microk8s-enable.wrapper" storage
+
+  # enable dns 
+  "$SNAP/microk8s-enable.wrapper" dns
+
+  # enable metrics-server
+  "$SNAP/microk8s-enable.wrapper" metrics-server
+   
+  run_with_sudo apt update
+
+  run_with_sudo apt install jq -y
+}
+
+
+
+enable_gopaddle_lite() {
+
+echo "Enabling gopaddle lite"
+# download gopaddle-lite helm
+
+# add helm repo
+run_with_sudo $HELM repo add gp-lite https://gopaddle-io.github.io/gopaddle-lite
+
+#list helm repo
+run_with_sudo $HELM repo update
+
+# create namespace
+if $KUBECTL get ns gp-lite >/dev/null 2>&1
+then
+ echo ""
+  echo "gp-lite namespace created."
+else
+  run_with_sudo $KUBECTL create ns gp-lite
+fi
+
+NODENAME=""
+
+NODELABEL=$($KUBECTL get node -l gp-install-node=node1 -o json | jq '.items | length' )
+
+if [ "$NODELABEL" = "0" ]
+then
+#getting node name
+NODENAME=$(run_with_sudo $KUBECTL get node -o json| jq '.items[0].metadata.name'|cut -d '"' -f 2)
+echo ""
+echo "Adding label 'gp-install-node=node1' to the node '$NODENAME'"
+echo ""
+#adding label in node 
+run_with_sudo $KUBECTL label nodes $NODENAME gp-install-node=node1 --overwrite=true
+else
+#getting node name
+NODENAME=$(run_with_sudo $KUBECTL get node -l gp-install-node=node1 -o json| jq '.items[0].metadata.name'|cut -d '"' -f 2)
+fi
+
+if [ $PUBLICIP == true ]
+then
+#install rabbitmq
+run_with_sudo $HELM install gp-rabbitmq-4-2 gp-lite/gp-lite  --namespace gp-lite  --set global.routingType=NodePortWithOutIngress --set global.installType=public --set global.storageClassName=$STORAGECLASS --set global.gp-rabbitmq.enabled=true --set global.gp-lite-core.enabled=false --set global.nodeSelector.gp-install-node=node1 --set global.nodeName=$NODENAME  --version $HELM_VERSION
+
+#install gp-lite
+sleep 1m
```

Comment:
Please do not use sleep like this here. Most people with think this addon is stuck ad will kill it. Instead:
- print out a message informing users why are we waiting, eg "Waiting for rabbitmq deployment to finish."
- use `kubectl rollout status` to wait for a resource you are interested in.